src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -129,11 +129,11 @@
   <data name="VersionOptionDescription" xml:space="preserve">
     <value>Version of the tool package in NuGet.</value>
   </data>
-  <data name="SourceFeedOptionDescription" xml:space="preserve">
+  <data name="AddSourceOptionDescription" xml:space="preserve">
     <value>Adds an additional NuGet package source to use during installation.</value>
   </data>
-  <data name="SourceFeedOptionName" xml:space="preserve">
-    <value>SOURCE_FEED</value>
+  <data name="AddSourceOptionName" xml:space="preserve">
+    <value>ADD_SOURCE</value>
```

Comment:
I think just `SOURCE` is fine.

Reply:
source will be confused with restore source (exclusive)

Reply:
I see what you saying. Let me change to source

Reply:
fixed

Reply:
Do we need another loc update?